content/practices/high-contrast/high-contrast-practice.html

```diff
@@ -0,0 +1,779 @@
+        </p>
+        <table class="data">
+          <caption>Invert Colors Example: Button Switch</caption>
+          <thead>
+            <tr>
+              <th>Setting</th>
+              <th>Screen Shot</th>
+            </tr>
+          </thead>
+          <tbody>
+            <tr>
+              <td>Invert Colors: Off (default)</td>
+              <td><img src="./images/currentcolor-macos-invert-off.png" alt="Screen shot of switch example with invert colors turned off" style="height: 85px"></td>
+            </tr>
+            <tr>
+              <td>Invert Colors: on</td>
+              <td><img src="./images/currentcolor-macos-invert-on.png" alt="Screen shot of switch example with invert colors turned on" style="height: 85px"></td>
+            </tr>
+          </tbody>
+        </table>
+      </section>
+
+      <section id="prefers-contrast">
+        <h2>Increase Contrast</h2>
+        <p>When the increase contrast setting is is enabled the <code>prefers-contrast</code> media query changes from <code>no-preference</code> to <code>more</code>.
+          The author <em>should</em> update the rendering of text content and components to meet
+          <a href="https://www.w3.org/WAI/WCAG21/Understanding/contrast-enhanced">WCAG 1.4.6: Contrast (Enhanced)</a>
+          requirement. 
+          If content already meets the enhanced requirement, the author <em>may</em> want to exceed the enhanced color contrast requirement.
+        </p>
+        <p>Note: Operating systems that support the <code>forced-colors</code> media query will set <code>prefers-contrast</code> to <code>custom</code> when the <code>forced-colors</code> is set to <code>active</code>.  </p>
+
+       <h3 id="prefers-contrast-example">Increase Contrast Example</h3>
+
+        <p>The example shows how the color contrast was increased from 12.8 to 18.1 for text and 4.6 to 12.2 for border and fill for the button when the <code>prefers-contrast: more</code> is set.  The example simulates the change on colors when the increased option switch is set to "on".  If you enable operating system "increased contrast" feature the example will use be rendered in any state with the increased contrast values.</p>
+
+        <div class="example">
+
+            <button type="button" role="switch" aria-checked="false" class="increase-contrast">
+              <span class="label">Increase Contrast</span>
+              <svg xmlns="http://www.w3.org/2000/svg" height="20" width="36">
+                <rect class="container" x="1" y="1" width="34" height="18" rx="9" />
+                <circle class="off" cx="11"  cy="10" r="8" />
+                <circle class="on"  cx="25" cy="10" r="8" />
+              </svg>
+              <span class="on" aria-hidden="true">On</span>
+              <span class="off" aria-hidden="true">Off</span>
+            </button>
```

Comment:
I’m not seeing any change to the page when I toggle either the Increase Contrast or Dark Color Scheme switches. Are those meant to immediately show a simulated effect?

Reply:
Yes, there are suppose to change.
It may be there should be a separate switch to tun on the emulation next to the example.

Reply:
Thanks @jongund, this is clearer. However, because the examples themselves _**are**_ toggle switches, I wonder if it might still invite a bit of confusion for some readers since the concepts of dark mode and high contrast are so often expressed as interactive toggles. 😅 

If the examples were just some static rich text, like a heading, a paragraph, and some sort of colored embellishment (like a blockquote or bulleted list), then I think they would be crystal clear — especially for the users who accept our invitation to enable their own device settings to observe the reaction.